docs/even-segmented-path-for-put-operation.md

```diff
@@ -0,0 +1,33 @@
+# EvenSegmentedPathForPutOperation
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Put-V1-02
+
+## Output Message
+
+API path with PUT operation defined MUST have even number of segments (i.e. end in {resourceType}/{resourceName} segments).
+
+## Description
+
+API path with PUT operation defined MUST have even number of segments (i.e. end in {resourceType}/{resourceName} segments).
+
+## How to fix the violation
+
+Fix the path with PUT defined as one of the below patterns:
+
+```json
+"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MyNameSpace/MyResourceType/{ResourceName}:
```

Comment:
>: [](http://example.com/codeflow?start=129&length=1)

: is intentional?

Reply:
but can remove it, if you think it isnt needed